htdocs/js/jquery.quickreply.js

```diff
@@ -45,6 +49,49 @@ function update(data,widget) {
     if ( ! customsubject || cur_subject == "" )
         subject.val(data.subject);
 
+    // If we previously munged the layout of #qrformdiv, reset it.
+    $('#qrformdiv').removeAttr('style').removeClass('width-adjusted');
+
+    // If replying to a comment, sort out the width. If the reply form would be
+    // super-small but there's plenty of whitespace to the left due to comment
+    // indentation, extend the form out to the left.
+    if ( data.target.match(/^\d+$/) ) {
```

Comment:
All this makes me wonder if a better UI is just to pop up a modal or something for comment entry when you click to reply. Is it actually important to us that the reply form end up embedded in the middle of the entry form?

Reply:
I think so. Or more specifically, the important part is being able to scroll up and down and remind yourself of the surrounding context as you're composing. Obscuring any part of the page would be iffy for that, and being a true modal that blocks scrolling would probably make people flip OUT. (I'd prolly flip out, at least.)

Reply:
Yeah, I don't mean a modal-modal, more just meant a frame of some kind that doesn't change the flow of the page. Definitely not worth touching right now but feels like it would be a more sustainable solution with all the different style madness things people can do. Food for thought I suppose.